modules/history/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@obsidianjs/history",
+  "version": "0.1.0",
+  "description": "History module allows you to manage an historic.",
```

Comment:
"Historic" is not the corresponding english word for the french "Historique"..   
=> https://en.wikipedia.org/wiki/Undo